src/components/LoadingCircle/LoadingCircle.tsx

```diff
@@ -0,0 +1,52 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { merge } from "@utilities/merge";
+import React, { FC } from "react";
+
+export enum LoadingCircleStyle {
+    Progress = "Progress",
+    Positive = "Positive",
+    Danger = "Danger",
+}
+
+export enum LoadingCircleSize {
+    ExtraSmall = "ExtraSmall",
+    Small = "Small",
+    Medium = "Medium",
+    Large = "Large",
+}
+
+export type LoadingCircleProps = {
+    style?: LoadingCircleStyle;
+    size?: LoadingCircleSize;
+};
+
+export const statusClasses: Record<LoadingCircleStyle, string> = {
+    [LoadingCircleStyle.Progress]: "tw-border-violet-60",
+    [LoadingCircleStyle.Positive]: "tw-border-green-60",
+    [LoadingCircleStyle.Danger]: "tw-border-red-60",
+};
+
+export const sizeClasses: Record<LoadingCircleSize, string> = {
+    [LoadingCircleSize.ExtraSmall]: "tw-w-[16px] tw-h-[16px]",
+    [LoadingCircleSize.Small]: "tw-w-[20px] tw-h-[20px]",
+    [LoadingCircleSize.Medium]: "tw-w-[32px] tw-h-[32px]",
+    [LoadingCircleSize.Large]: "tw-w-[64px] tw-h-[64px]",
```

Comment:
```suggestion
    [LoadingCircleSize.ExtraSmall]: "tw-w-4 tw-h-4",
    [LoadingCircleSize.Small]: "tw-w-5 tw-h-5",
    [LoadingCircleSize.Medium]: "tw-w-8 tw-h-8",
    [LoadingCircleSize.Large]: "tw-w-16 tw-h-16",
```
We can avoid more CSS being generated by using those